.ci/.storybook/main.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+const config = require('@kbn/storybook').defaultConfig;
+const aliases = require('../../src/dev/storybook/aliases.ts').storybookAliases;
+
+config.refs = {};
+
+for(const alias of Object.keys(aliases).filter(a => a !== 'ci_composite')) {
+  // snake_case -> Title Case
+  const title = alias
```

Comment:
In a draft PR (#80061) I started by making the aliases an object so that a `title` could be added. Doing this transformation would probably be fine if there were no title specified, but it would be better to allow a title. For instance here "apm" would end up as "Apm" where we want APM.